src/_theme.scss

```diff
@@ -0,0 +1,16 @@
+@mixin trans-background($color, $opacity: 0.9) {
+  background-color: $color;
+  background-color: rgba($color, $opacity);
+}
+
+$purple: #7d5cc6;
+$dark-gray: #464646;
+$gray: #333333;
+$light-gray: #d2d2d2;
+$yellow: #f3cf45;
```

Comment:
I feel this is a strange combination of color. Purple, gray, yellow

Reply:
This is the color scheme on CCNY website.